<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="logo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>231</width>
      <height>191</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#logo {
background-color: transparent;
border-image: url(:/logo.PNG);
background: none;
border: none;
background-repeat: none;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="clients">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#clients {
    background-color: #FFE7E7;
    border-style: outset;
    
    border-color: #00000;
    font: bold 16px;
    color:#00000;
    height:10px;
       padding: 2px;
}</string>
    </property>
    <property name="text">
     <string>Gestion des clients</string>
    </property>
   </widget>
   <widget class="QPushButton" name="evenements">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#evenements {
    background-color: #FFE7E7;
    border-style: outset;
    
    border-color: #00000;
    font: bold 16px;
    color:#00000;
    height:10px;
       padding: 2px;
}</string>
    </property>
    <property name="text">
     <string>Gestion des évènements</string>
    </property>
   </widget>
   <widget class="QPushButton" name="espaces">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#espaces {
    background-color: #FFE7E7;
    border-style: outset;
    
    border-color: #00000;
    font: bold 16px;
    color:#00000;
    height:10px;
       padding: 2px;
}</string>
    </property>
    <property name="text">
     <string>Gestion des espaces</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fournisseurs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#fournisseurs {
    background-color: #FFE7E7;
    border-style: outset;
    
    border-color: #00000;
    font: bold 16px;
    color:#00000;
    height:10px;
       padding: 2px;
}</string>
    </property>
    <property name="text">
     <string>Gestion des fournisseurs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="emplyes">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#emplyes {
    background-color: #FFE7E7;
    border-style: outset;
    
    border-color: #00000;
    font: bold 16px;
    color:#00000;
    height:10px;
       padding: 2px;
}</string>
    </property>
    <property name="text">
     <string>Gestion des employés</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>781</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="ajouter">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QPushButton" name="quitter">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px;
font: bold;</string>
      </property>
      <property name="text">
       <string>Adresse email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px;
font: bold;</string>
      </property>
      <property name="text">
       <string>Numéro de téléphone</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px;
font: bold;</string>
      </property>
      <property name="text">
       <string>Date de naissance</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px;
font: bold;</string>
      </property>
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px;
font: bold;</string>
      </property>
      <property name="text">
       <string>cin</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px;
font: bold;</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ad">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>230</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="cin">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenom">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>70</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="num_tel">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>190</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouterr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="date">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>150</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="pic">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>50</y>
        <width>261</width>
        <height>341</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="photo">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>420</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>insert</string>
      </property>
     </widget>
     <widget class="QPushButton" name="aff">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>50</y>
        <width>241</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#aff{
background-color: transparent;
background: none;
border: none;
background-repeat: none;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px;
font: bold;</string>
      </property>
      <property name="text">
       <string>Sexe</string>
      </property>
     </widget>
     <widget class="QComboBox" name="sexe">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>120</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>homme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>femme</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="afficher">
     <attribute name="title">
      <string>Afficher</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>751</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px;
font: bold;</string>
      </property>
      <property name="text">
       <string>cin</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cin_aff">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QComboBox" name="tri">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>41</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>prenom par ordre croissant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>prenom par ordre decroissant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nom par ordre croissant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>nom par ordre decroissant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>cin par ordre croissant</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>cin par ordre decroissant</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px;
font: bold;</string>
      </property>
      <property name="text">
       <string>Trier par:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>pdf</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="supprimer_2">
     <attribute name="title">
      <string>Supprimer</string>
     </attribute>
     <widget class="QComboBox" name="cin_sup">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimer">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>170</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="stati">
     <attribute name="title">
      <string>Statistiques</string>
     </attribute>
     <widget class="QPushButton" name="stat">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>stat</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>491</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>Il es possible de génerer des statistiques en cliquant sur le bouton statististiques:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="vip">
     <attribute name="title">
      <string>VIP</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 14px;
font: bold;</string>
      </property>
      <property name="text">
       <string>cin</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cin_qr">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="qr_gv">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>461</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1173</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
